File: TypeScript/ex1.ts
function ex1() : void
{
    let tab: number[][] = [
        [0, 0, 0],
        [0, 1, 0],
        [0, 0, 0]
    ];
    for (let i = 0; i < 3; i++)
    {
        console.log("-------------");
        let variable: String = "";
        for (let y = 0; y < 3; y++)
        {
            variable += "| " + tab[i][y] + " ";
        }
        console.log(variable+"|");
        
    }
    console.log("-------------");
    return;
}

ex1();

File: rust/src/ex1.rs
pub fn ex1(){
    let tab: [[i32; 3]; 3] = [[0,0,0],[0,1,0],[0,0,0]];//[[0; 4]; 3]

    for i in 0..3 {
        println!("-------------");
        for y in 0..3 {
            print!("| {} ",tab[i][y]);
        }
        println!("|");
    }
    println!("-------------");
    return ;
}
